packages/amplify-graphiql-explorer/config/env.js

```diff
@@ -0,0 +1,104 @@
+'use strict';
```

Comment:
Almost all the changes in this PR come from running CRA's eject script, which is necessary to add the polyfills needed in this package.